src/app/components/common/node-editor/node-editor.component.ts

```diff
@@ -182,7 +183,7 @@ export class NodeEditorComponent implements OnInit {
       resetNodes();
     }
     if (event.key === 'Delete') {
-      this.deleteSelectedConnection();
+      this.deleteSelected();
```

Comment:
deleteSelectedNodes?

Reply:
This method is for both `deleteSelectedNodes` and `deleteSelectedConnection`.